test/services/groups/transfer_service_test.rb

```diff
@@ -4,82 +4,39 @@
 
 module Groups
   class TransferServiceTest < ActiveSupport::TestCase
+    include ActiveJob::TestHelper
```

Comment:
As per the general comment on the PR this can be moved into `test/test_helper.rb`

Reply:
Please see 0ef0afd6a0e156a6483e4434ea25dde01d85eebd.